src/app/pages/administration/jobs/system-trigger-create.jsx

```diff
@@ -18,16 +18,16 @@ import theme from 'themes/theme';
 
 const useStyles = makeStyles({
   container: {
-    display: 'grid',
-    gap: '16px',
-    gridTemplateColumns: '1fr',
+    display: 'flex',
+    flexDirection: 'column',
     [theme.breakpoints.up('sm')]: {
-      gridTemplateColumns: '1fr 1fr',
+      display: 'grid',
+      gap: '16px',
+      gridTemplateColumns: '1fr',
```

Comment:
Do we even need this class? The only thing inside the element that is `className = container` is a single `<div>` now (though maybe should be just a `<Card>`). Is there any layout that can be done with a single element?